src/integration/java/org/springframework/data/couchbase/IntegrationTestApplicationConfig.java

```diff
@@ -20,35 +18,31 @@
 @Configuration
 public class IntegrationTestApplicationConfig extends AbstractCouchbaseConfiguration {
 
-  @Autowired
-  private Environment springEnv;
-
   @Bean
   public String couchbaseAdminUser() {
-    return springEnv.getProperty("couchbase.adminUser", "Administrator");
```

Comment:
is it possible to not hardcode but then also use the spring env property settings in the docker container startup? that way it remains configurable

Reply:
The configuration for docker containers has predefined values and already done before Couchbase configuration. I see no specific benefit of making it configurable as the preferred way is to use containers, however it can be changed in code for debugging.